### ISSUE TITLE
jet set radio graphics are broken on gfxgl3

### ISSUE THREAD

**Issue by snickerbockers** [washingtondc-emu/washingtondc#110]
```
f48a0394bd4194288a526af71a65f3cfa132e452 is the first bad commit
commit f48a0394bd4194288a526af71a65f3cfa132e452
Author: snickerbockers <snickerbockers@washemu.org>
Date:   Mon Nov 23 12:53:09 2020 -0500

    pvr2: the clip limits are now stored in the display lists and calculated by the TA

 src/libwashdc/hw/pvr2/pvr2_core.c | 96 ++++-----------------------------------
 src/libwashdc/hw/pvr2/pvr2_core.h | 13 +++---
 src/libwashdc/hw/pvr2/pvr2_ta.c   | 75 ++++++++++++++++++++++++++++++
 3 files changed, 92 insertions(+), 92 deletions(-)
```
![2022-12-22-20-00-13](https://user-images.githubusercontent.com/3129682/209250668-71cbc146-2e7e-44e2-9d57-b05b457073e3.png)

**Comment by snickerbockers** [comment]
at commit 46e144da26baa680de37c4f32659ee10bd90a6ba:
```
GFX_IL COMMAND GFX_IL_SET_CLIP_RANGE
GFX_IL	clip_min 0.000000
GFX_IL	clip_max 2.358087
```

at commit f48a0394bd4194288a526af71a65f3cfa132e452:
```
GFX_IL COMMAND GFX_IL_SET_CLIP_RANGE
GFX_IL	clip_min 0.000000
GFX_IL	clip_max 100000.000000
```

**Comment by snickerbockers** [comment]
this issue has been resolved by commit e4be1a586e1997050182010394ce1248a3c79eab
